Tangem/Modules/Markets/TokenList/MarketsItemViewModel.swift

```diff
@@ -92,7 +95,22 @@ class MarketsItemViewModel: Identifiable, ObservableObject {
 
     private func setupPriceInfo(price: Decimal?, priceChangeValue: Decimal?) {
         priceValue = priceFormatter.formatFiatBalance(price)
-        priceChangeState = priceChangeUtility.convertToPriceChangeState(changePercent: priceChangeValue)
+        priceChangeState = priceChangeUtility.convertToPriceChangeState(changePercent: priceChangeValue, isDevide: false)
+    }
+
+    private func bindToIntervalUpdates() {
+        filterProvider?.filterPublisher
+            .dropFirst()
+            .removeDuplicates()
+            .withWeakCaptureOf(self)
+            .sink { viewModel, filter in
+                guard let newQuote = viewModel.quotesRepository.quotes.first(where: { $0.key == viewModel.tokenId })?.value else {
+                    return
+                }
```

Comment:
там же словарь, не надо в цикле гонять
```suggestion
                guard let newQuote = viewModel.quotesRepository.quotes[viewModel.tokenId] else {
                    return
                }
```

Reply:
> там же словарь, не надо в цикле гонять

Угу неверный рудимент разработки